Tests/TestSupport/Mocks/MockEmbraceConfigurable.swift

```diff
@@ -92,11 +92,11 @@ public class MockEmbraceConfigurable: EmbraceConfigurable {
         }
     }
 
-    public let updateExpectation = XCTestExpectation(description: "update called")
+    public var updateCallCount = 0
```

Comment:
Curious why you moved away from the XCTestExpectation. Was this causing test failures?